source/nsdynmemLIB/nsdynmemLIB.c

```diff
@@ -117,77 +115,51 @@ void dev_stat_update(mem_stat_update_t type, int16_t size)
     }
 }
 
-//Fix return value allways good fo aligment
+//Fix return value to be always good for alignment
```

Comment:
Bit misleading - it returns number of sectors, rather than an aligned byte size. Although that's far more obvious from the clearer code anyway.